docs/SETUP.md

```diff
@@ -188,4 +190,5 @@ QUB students should follow the [Discord Bot guidelines](../raspberry/DiscordBot)
 
 ## You are now ready to fly, go to [How to fly](../README.md#how-to-fly) 
 
-[^1]: Remember you can use the [DiscordBot](../raspberry/DiscordBot) to find the Quadcopters IP Address
+[^1]: Remember you can use the [DiscordBot](../raspberry/DiscordBot) to find the Quadcopters username and hostname
+[^2]: Remember you can use the [DiscordBot](../raspberry/DiscordBot) to find the Quadcopters IP Address
```

Comment:
Quadcopter's or Quadcopters'

Reply:
ChatGPT 

The correct grammar depends on the context, similar to the previous example.
If you are referring to a single quadcopter and its IP address, you would use "The quadcopter's IP address" because you're indicating possession by a singular noun ("quadcopter").

Reply:
Yes, that's what I meant - but you need an apostrophe.

Reply:
Yea I fixed thanks